official/benchmark/benchmark_uploader.py

```diff
@@ -132,3 +132,19 @@ def _upload_json(self, dataset_name, table_name, json_list):
     if errors:
       tf.logging.error(
           "Failed to upload benchmark info to bigquery: {}".format(errors))
+
+  def insert_run_status(self, dataset_name, table_name, run_id, run_status):
+    """Insert the run status in to Bigquery run status table."""
+    query = ("INSERT {ds}.{tb} "
+             "(run_id, status) "
+             "VALUES('{rid}', '{status}')").format(
+                 ds=dataset_name, tb=table_name, rid=run_id, status=run_status)
+    self._bq_client.query(query=query).result()
```

Comment:
Does the result need to be validated at all? Or it will raise an error on failure? In the case of an error, presumably the  whole run will fail. Is that the desired behavior?